DigitalLearningSolutions.Web/ViewModels/Common/PasswordViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Common
+{
+    using System.ComponentModel.DataAnnotations;
+
+    public class PasswordViewModel
+    {
+        [Required(ErrorMessage = "Enter a password.")]
+        [MinLength(8, ErrorMessage = "Password must be at least 8 characters.")]

```

Comment:
Do we need any max length constraints (thinking about db constraints)?

Reply:
Since passwords are hashed and truncated we don't need a limit on them as far as the db is concerned, but other fields probably do. We should also enforce size limits on everything to make sure we don't go over 4096 bytes in TempData since that's the maximum size of a cookie.